docs/scripts_md/MRIProcessingUtility.md

```diff
@@ -261,13 +277,17 @@ Register programs.
 
 INPUT: program to register
 
-### moveAndUpdateTarchive($tarchive\_location, $tarchiveInfo)
+### moveAndUpdateTarchive($tarchive\_location, $tarchiveInfo, $upload\_id)
 
-Moves and updates the tarchive table with the new location of the tarchive.
+Moves and updates the `tarchive` table with the new location of the
+`tarchive`.
 
-INPUTS: tarchive location, tarchive information hashref
+INPUTS:
+  - $tarchive\_location: `tarchive location`,
+  - $tarchiveInfo     : `tarchive` information hashref,
+  - $upload\_id        : upload ID of the study
```

Comment:
not important but I know you care about these,  so just wanted to let you know, but ignore if you wish.
not sure why the `:` are not aligned (same for some functions below this as well).

Reply:
weird. It looks like that only in the md code. However, the MD file in the browser looks OK (it always ignores spaces anyway). I checked the .pm and all looked aligned so should be OK. Thanks for checking, my inner OCD thanks you ;).